src/elements/content-sidebar/activity-feed/SelectableActivityCard.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../common/variables';
+
+@keyframes bcs-SelectableActivityCard-hover {
+    from {
+        background-color: transparent;
+    }
+
+    to {
+        background-color: $bdl-box-blue-05;
+    }
+}
+
+.bcs-SelectableActivityCard {
+    &[aria-disabled='false'] {
+        cursor: pointer;
+
+        &:hover,
+        &:focus {
+            outline: none;
+            box-shadow: 0 0 8px 0 rgba(114, 132, 142, .16);
+            animation: bcs-SelectableActivityCard-hover 300ms ease-in-out 25ms 1 normal forwards;
```

Comment:
I'll address this along with https://github.com/box/box-ui-elements/pull/2508/files#r604374585 in a separate PR since I want the `bcs-is-focused` animation to override this one for now